indy_common/test/types/test_get_rs_schema_schema.py

```diff
@@ -0,0 +1,39 @@
+from collections import OrderedDict
+from indy_common.types import ClientGetRsSchemaOperation, RsSchemaMetaField
+from plenum.common.messages.fields import ConstantField, IdentifierField, VersionField, LimitedLengthStringField
+
+
+EXPECTED_ORDERED_FIELDS_SCHEMA = OrderedDict([
+    ("type", ConstantField),
```

Comment:
I think it would be rather to use constants instead of strings. For example, TYPE constant instead of "type" string.

Reply:
I originally had constants but as @ashcherbakov pointed out to me, the tests should catch accidental API changes. I prefer tests not having dependances on constants. If you strongly disagree I change tests to use constants.